x11-plugins/wmcdplay/wmcdplay-1.1.ebuild

```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
-inherit autotools
+inherit autotools desktop
 
 DESCRIPTION="CD player applet for WindowMaker"
 HOMEPAGE="https://www.dockapps.net/wmcdplay"
```

Comment:
ther can default be used from src_install()

Reply:
I've updated the PR, now i'm using ```default``` in ```src_install``` (also needed to set ```DOCS``` in order to get the right files installed)